UKHO.ADM.Tests/functions/Update-AD.Tests.ps1

```diff
@@ -2,7 +2,15 @@ InModuleScope $mut {
     Describe "Update-AD" {
         Context "with valid cd" {
             Mock Generate-ConfigurationObject       
-            Mock Get-ADChanges { return @{ "CreatedOUs" = 1}}
+            Mock Get-ADChanges { return @{ "CreatedOUs" = 1; SystemColours = @{
+                "info" = "White";
+                "adding" = "Green";
+                "remove" = "Red";
+                "modify" = "Yellow";
+                "error" = "Magenta";
+                "header" = "Blue";
```

Comment:
`header` isn't a great name. Alert? I wouldn't be sure what to use header for and would probably avoid using it.

Reply:
`section`? not sure it is an alert, it is a bit saying I'm Going to do this...

Reply:
`start`
`process`

Reply:
`progress`
`stage`
`phase`

Reply:
header is now `process`